_targets.R

```diff
@@ -7,14 +7,26 @@ options(tidyverse.quiet = TRUE)
 tar_option_set(packages = c("tidyverse", "dataRetrieval")) # Loading tidyverse because we need dplyr, ggplot2, readr, stringr, and purrr
 
 p1_targets_list <- list(
-  tar_target(
-    site_data,
-    download_nwis_data(),
+  tar_target(site_data_01427207, download_nwis_site_data('01427207')),
+  tar_target(site_data_01432160, download_nwis_site_data('01432160')),
+  tar_target(site_data_01436690, download_nwis_site_data('01436690')),
+  tar_target(site_data_01466500, download_nwis_site_data('01466500')),
+  
+  tar_target(site_data,
+             {
+               out_file <- "1_fetch/out/site_data.csv"
+               list(site_data_01427207, site_data_01432160, 
+                    site_data_01436690, site_data_01466500) %>% 
+                 bind_rows() %>% 
+                 write_csv(out_file)
+               return(out_file)
+             },
+             format = "file"
   ),
+  
   tar_target(
     site_info_csv,
-    nwis_site_info(fileout = "1_fetch/out/site_info.csv", site_data),
-    format = "file"
+    nwis_site_info(site_data, fileout = "1_fetch/out/site_info.csv")
```

Comment:
If you update to `site_data_csv` as suggested above, this would also need to change.